homeassistant/components/zha/strings.json

```
@@ -529,5 +529,403 @@
         }
       }
     }
+  },
+  "entity": {
+    "alarm_control_panel": {
+      "alarm_control_panel": {
+        "name": "[%key:component::alarm_control_panel::title%]"
+      }
+    },
+    "binary_sensor": {
+      "accelerometer": {
+        "name": "Accelerometer"
+      },
+      "binary_input": {
+        "name": "Binary input"
+      },
+      "frost_lock": {
+        "name": "Frost lock"
+      },
+      "replace_filter": {
+        "name": "Replace filter"
+      },
+      "consumer_connected": {
+        "name": "Consumer connected"
+      },
+      "valve_alarm": {
+        "name": "Valve alarm"
+      },
+      "calibrated": {
+        "name": "Calibrated"
+      },
+      "external_sensor": {
+        "name": "External sensor"
+      },
+      "linkage_alarm_state": {
+        "name": "Linkage alarm state"
+      },
+      "ias_zone": {
+        "name": "IAS zone"
+      }
+    },
+    "button": {
+      "reset_frost_lock": {
+        "name": "Frost lock reset"
+      },
+      "reset_no_presence_status": {
+        "name": "Presence status reset"
+      },
+      "feed": {
+        "name": "Feed"
+      },
+      "self_test": {
+        "name": "Self-test"
+      }
+    },
+    "climate": {
+      "thermostat": {
+        "name": "[%key:component::climate::entity_component::_::name%]"
+      }
+    },
+    "cover": {
+      "cover": {
+        "name": "[%key:component::cover::title%]"
+      },
+      "shade": {
+        "name": "[%key:component::cover::entity_component::shade::name%]"
+      },
+      "keen_vent": {
+        "name": "Keen vent"
+      }
+    },
+    "fan": {
+      "fan": {
+        "name": "[%key:component::fan::entity_component::_::name%]"
```

```suggestion
        "name": "[%key:component::cover::title%]"
```
Lokalise UI doesn't really like translation references deeper than 1 key